homeassistant/components/fronius/sensor.py

```diff
@@ -150,16 +159,16 @@ async def fetch_data(*_):
 class FroniusAdapter:
     """The Fronius sensor fetching component."""
 
-    def __init__(self, bridge, name, device, add_entities):
+    def __init__(self, bridge: Fronius, name, device, add_entities):
         """Initialize the sensor."""
         self.bridge = bridge
         self._name = name
         self._device = device
-        self._fetched = {}
+        self._fetched: dict = {}
```

Comment:
Can we make `dict` more specific?

Reply:
Done. I tried to type with "Dict[...]" but that was changed to "dict[...]" which I assumed is incorrect syntax. Seems to work though?

Reply:
This is Python 3.10 syntax which works here because of `from __future__ import annotations`. 
`dict[str, Any]` would fit I think, but isn't much more helpful than `dict`.